resync/api/monitoring_dashboard.py

```
@@ -43,506 +91,401 @@ class MetricSample:
+            redis = get_redis_client()
+            pipe = redis.pipeline()
+            for alert in new_alerts:
+                pipe.lpush(REDIS_KEY_ALERTS, json_dumps(alert))
+            pipe.ltrim(REDIS_KEY_ALERTS, 0, 19) # Manter últimos 20
+            await pipe.execute()
+
+    async def get_global_uptime(self) -> float:
+        redis = get_redis_client()
+        try:
+            now = time.time()
+            was_set = await redis.set(REDIS_KEY_START_TIME, str(now), nx=True)
```

Ruff (0.15.0)

[error] Local variable `was_set` is assigned to but never used

Remove assignment to unused variable `was_set`

(F841)